### DISCUSSION TITLE
Orca slicer introduces unwanted M600 filament change command at start of print.

### DISCUSSION THREAD

**Discussion by RemyLebeau** [OrcaSlicer/OrcaSlicer#6140] [Q&A] [answered]
Whenever I do a print that has a mid print filament change, the slicer also introduces a filament change command at the very start of the print.  I would like it to not do that!

If I slice an object with no filament changes, there are no M600 commands in the GCODE.

If I slide the same object with a filament change say half way, there are TWO M600 commands added - one at the very start, and then next at the layer I specify.

Is there a way to prevent this first M600 from being added?  All it does is add another trip to the printer, and waste filament.

**Comment by RemyLebeau** [comment, marked as answer]
It's already listed as a bug: https://github.com/SoftFever/OrcaSlicer/issues/6089

**Comment by zetabemol** [comment]
Debes tildar (configuracion de la impresora/Multimaterial) el cambio de filamento manual. Sino lo haces lo interpreta como un cambio de herramienta y te genera un m600 al iniciar el gcode
